includes/database/schemas/class-order-adjustments.php

```diff
@@ -69,9 +69,18 @@ class Order_Adjustments extends Schema {
 			'type'       => 'bigint',
 			'length'     => '20',
 			'unsigned'   => true,
-			'default'    => '0',
+			'default'    => null,
 			'searchable' => true,
-			'sortable'   => true
+			'sortable'   => true,
```

Comment:
```suggestion
			'sortable'   => true,
			'allow_null' => true,
```

Required to allow a `null` value.